contract/src/lib.rs

```diff
@@ -296,4 +296,15 @@ impl MpcContract {
 
     #[allow(unused_variables)]
     pub fn respond(&mut self, receipt_id: [u8; 32], big_r: String, s: String) {}
+
+    #[private]
+    #[init(ignore_state)]
+    pub fn clean(keys: Vec<near_sdk::json_types::Base64VecU8>) -> Self {
+        for key in keys.iter() {
+            env::storage_remove(&key.0);
```

Comment:
where is clean being called from? I wonder if we should return an error just in case `storage_remove` couldn't find the respective key

Reply:
Just called externally via NEAR CLI, like in [this](https://gist.github.com/ilyar/19bdc04d1aa09ae0fc84eb4297df1a1d) guide